src/librustc_mir/repr.rs

```diff
@@ -310,18 +310,13 @@ impl<'tcx> Debug for Terminator<'tcx> {
     fn fmt(&self, fmt: &mut Formatter) -> Result<(), Error> {
         use self::Terminator::*;
         match *self {
-            Goto { target } =>
-                write!(fmt, "goto -> {:?}", target),
-            Panic { target } =>
-                write!(fmt, "panic -> {:?}", target),
-            If { cond: ref lv, ref targets } =>
-                write!(fmt, "if({:?}) -> {:?}", lv, targets),
+            Goto { target } => write!(fmt, "goto -> {:?}", target),
+            Panic { target } => write!(fmt, "panic -> {:?}", target),
+            If { cond: ref lv, ref targets } => write!(fmt, "if({:?}) -> {:?}", lv, targets),
             Switch { discr: ref lv, ref targets } =>
                 write!(fmt, "switch({:?}) -> {:?}", lv, targets),
-            Diverge =>
-                write!(fmt, "diverge"),
-            Return =>
-                write!(fmt, "return"),
+            Diverge => write!(fmt, "diverge"),
+            Return => write!(fmt, "return"),
```

Comment:
Do we care about consistency here? I usually try to make the arms either all the same, or else have a block of really-short, one-line arms followed by longer, indented arms. But maybe this is OK. (Note that I feel differently when there are a bunch of `|` patterns.)